### DISCUSSION TITLE
Android AR button disappears on click

### DISCUSSION THREAD

**Discussion by goldenpath-ai** [google/model-viewer#2224] [Q&A]
### Description

When viewing the model on an Android device and clicking on the AR button, the AR button disappears instead of launching scene viewer. Works as it should on iOS devices.

### Version

- model-viewer: v1.2.1

### Browser Affected

- [ x ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ x ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
This is unfortunately a known issue. We have no way to determine for sure if a phone is completely AR-capable until we try. If it fails at that point, then we remove the button. Does the same thing happen with webXR? Have you tried changing the `ar-modes`? And does sceneViewer work on this phone under other circumstances? Also, what version of Android are you on? We have several issues we're tracking regarding Android 11.
